### PR TITLE
Move ci deploy to eu-central-1

### PR DESCRIPTION
**Pull request by nustiueudinastea**

### CODE REVIEW

**Comment by nustiueudinastea** [comment]
Fixes https://github.com/microservices-demo/microservices-demo/issues/600

File: deploy/docker-swarm/infra/aws/variables.tf [hunk 1]
```diff
@@ -1,4 +1,4 @@
-variable "region" {
+variable "aws_region" {
   default = "eu-west-1"
```
Comment (vlal-zz) [inline deploy/docker-swarm/infra/aws/variables.tf]: should this also be changed to eu-central-1?

Comment (jasonrichardsmith) [reply deploy/docker-swarm/infra/aws/variables.tf]: agreed

**Review by jasonrichardsmith — APPROVED** [review]

File: deploy/docker-swarm/infra/aws/variables.tf (hunk 1 above)
Comment (nustiueudinastea) [reply deploy/docker-swarm/infra/aws/variables.tf]: I wil change it, although it's not strictly required, because we set it with an environment variable.

**Pull request merged**